app/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
If proguard is not used, let's get rid of this file.

Reply:
Agree. Removed.